src/Parlot/Fluent/ParseContext.cs

```diff
@@ -43,7 +43,7 @@ public ParseContext(Scanner scanner, bool useNewLines = false)
 
         public void SkipWhiteSpace()
         {
-            if (WhiteSpaceParser is null)
+            if (WhiteSpaceParser == null)
```

Comment:
not relevant here, but `is null` is better that way that it translates directly to `ReferenceEquals(x, null)` instead jumping into overloaded equality operator if it's present, Jint says Hi